RSignatureVerifier/Bundle+Environment.swift

```diff
@@ -0,0 +1,40 @@
+extension Bundle: EnvironmentSetupProtocol {
+    var valueNotFound: String {
+        return "NONE"
+    }
+
+    func osVersion() -> String {
```

Comment:
It can be a computed property:
var osVersion: String {...}

Reply:
done